src/framework/midi/miditypes.h

```diff
@@ -135,8 +135,6 @@ struct MidiDevice {
     }
 };
 
-using MidiDeviceList = std::vector<MidiDevice>;

```

Comment:
can you explain why you decided to delete the alias?

Reply:
can you explain in more detail? because I don't agree with you in this case

Reply:
It isn't a list but a vector. And the alias makes it opaque, you dont know what methods you can use on it.
Anyway this commit is purely cosmetic, so I can withdraw it.

Reply:
yes, please